src/main/java/jp/ikedam/jenkins/plugins/jobcopy_builder/JobcopyBuilder.java

```diff
@@ -232,7 +232,7 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
         
         if(fromJob == null)
         {
-            listener.getLogger().println(String.format("Error: Item '%s 'was not found.", fromJob));
+            listener.getLogger().println(String.format("Error: Item '%s 'was not found.", fromJobNameExpanded));

```

Comment:
<BugInstance type='NP_LOAD_OF_KNOWN_NULL_VALUE' priority='Normal' category='STYLE' message='Load of known null value in jp.ikedam.jenkins.plugins.jobcopy_builder.JobcopyBuilder.perform(AbstractBuild, Launcher, BuildListener)' lineNumber='235'/>